### PR TITLE
Feature/s4 79/wait to sign confirmation

### PR DESCRIPTION
**Pull request by sapphireduffy**
#<Feature Title>

-[x] 3 Amigos
-[x] Acceptance Criteria met
-[x] Pulled Master Branch
-[x] Manually Tested 
-[x] Reviewed Code
-[x] Docker-compose environment variables are consistent with those on rebel1

##Testing
-[x] Tested manually in browser
-[x] Added Unit Tests and they are passing
-[x] WAVE Accessibility Testing

##Screenshots of views

Screenshot 2020-07-17 at 13 06 42

### CODE REVIEW

File: src/views/wait-for-others-to-sign.njk [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        <p class="govuk-body govuk-!-font-weight-bold">Those signing must:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>select the link and start to go through the service</li>
+          <li>sign in to or register for a Companies House account using the email address that you have given us in this service</li>
+          <li>have the company's authentication code in order to sign</li>
+        </ul>
+
+        <p class="govuk-body">When all directors or members have signed you will be able to pay for and submit the application.</p>
+        <p class="govuk-body">You can now exit the page</p>
```
Comment (mateuszc-kainos) [inline src/views/wait-for-others-to-sign.njk:37]: Should say "this page"

File: src/views/wait-for-others-to-sign.njk [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
```
Comment (mateuszc-kainos) [inline src/views/wait-for-others-to-sign.njk:4]: Do we need that?

File: src/views/wait-for-others-to-sign.njk [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{% endblock %}
+
+{% block content %}
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <div class="govuk-panel" style="background:#1d70b8">
+          <h1 class="govuk-panel__title govuk-heading-l" style="color:#ffffff">
+             You must ask the other directors or members to sign the application
```
Comment (mateuszc-kainos) [inline src/views/wait-for-others-to-sign.njk:18]: Just a heads up - it was my call to use plain html here as the existing govuk component didn't allow for customization. Suggestions to make it cleaner are welcome

File: src/views/wait-for-others-to-sign.njk (hunk 2 above)
Comment (sapphireduffy) [reply src/views/wait-for-others-to-sign.njk]: Just tested it - we don't need it!

File: src/views/wait-for-others-to-sign.njk (hunk 3 above)
Comment (peadarkelly) [reply src/views/wait-for-others-to-sign.njk:17]: Inline CSS :drake-no: 

I'm pretty sure appeals added a similar component here https://github.com/companieshouse/lfp-appeals-frontend/blob/master/src/views/other-reason-disclaimer.njk#L22

If absolutely necessary, custom CSS can be added to https://github.com/companieshouse/cdn.ch.gov.uk

Try reaching out to the Tiny Rebels to see how they went about creating their component

File: src/views/view-final-confirmation.njk [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{% extends 'layout.njk' %}
```
Comment (peadarkelly) [inline src/views/view-final-confirmation.njk:1]: I don't think this page should be here...

File: test/controllers/waitForOthersToSign.test.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import { createApp } from './helpers/application.factory'
+import HtmlAssertHelper from './helpers/htmlAssert.helper'
+
+import 'app/controllers/waitForOthersToSign.controller'
+import { WAIT_FOR_OTHERS_TO_SIGN_URI } from 'app/paths'
+
+describe('WaitForOthersToSignController', () => {
+  describe('GET request', () => {
+    it('should render the WaitForOthers page', async () => {
+        const app = createApp()
```
Comment (peadarkelly) [inline test/controllers/waitForOthersToSign.test.ts:15]: indent is inconsistent. Make sure to update your vscode preferences to 2-space indent

File: src/views/wait-for-others-to-sign.njk (hunk 3 above)
Comment (sapphireduffy) [reply src/views/wait-for-others-to-sign.njk:17]: Ok thanks everyone, reached out to Juliano to ask how they created a component!

File: src/views/wait-for-others-to-sign.njk [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  <main class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <div class="interruption-card">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">You must ask the other directors or members to sign the application</h1>
```
Comment (mateuszc-kainos) [inline src/views/wait-for-others-to-sign.njk:16]: Please add id to header for ease of UI testing. 'interruption-card-header' should suffice

File: src/views/wait-for-others-to-sign.njk [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+
+    <div class="govuk-grid-row govuk-!-margin-top-4">
+      <div class="govuk-grid-column-two-thirds">
+
+        <p class="govuk-body govuk-!-font-weight-bold">Before submitting, you must now:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>email the other directors or members or those signing for them to ask them to sign the application</li>
+          <li>give them the link to the Apply to strike off or dissolve a company service: <a href="#" class="govuk-link">http://guidedfiling.companieshouse.gov.uk/start?t=DS01</a>
+          </li>
```
Comment (peadarkelly) [inline src/views/wait-for-others-to-sign.njk:28]: move this onto the previous line

File: test/controllers/waitForOthersToSign.test.ts [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+     const res = await request(app)
+      .get(WAIT_FOR_OTHERS_TO_SIGN_URI)
+      .expect(OK)
+
+     const htmlAssertHelper: HtmlAssertHelper = new HtmlAssertHelper(res.text)
+
+     assert.isTrue(htmlAssertHelper.hasText('h1', 'You must ask the other directors or members to sign the application'))
+      })
+    })
```
Comment (peadarkelly) [inline test/controllers/waitForOthersToSign.test.ts:25]: indentation is off

**Pull request merged**